### PR TITLE
Git: update Husky pre-commit for POSIX sh Compatibility

### PR DESCRIPTION
**Pull request by dcramer**
Removes the use of double brackets, which is not compatible with all shell implementations.

### CODE REVIEW

**Comment by dcramer** [comment]
It escapes me what the actual difference is (I'm by no means a shell scripting expert), but this was throwing parse errors.

File: .husky/pre-commit [hunk 1]
```diff
@@ -5,19 +5,19 @@
 cd "$(dirname -- "$0")/.." || exit 1
 
 # Ensure node is available (same as asciinema-rec.sh)
-if [[ ! -x "$(command -v node)" ]]; then
-  if [[ ! -x "$(command -v nvm)" && -f ~/.nvm/nvm.sh ]]; then
+if [ ! -x "$(command -v node)" ]; then
```
Comment (dcramer) [inline .husky/pre-commit:8]: aside you could run volta these days if you werent aware:

https://volta.sh/

makes managing a lot of this _so much easier_

File: .husky/pre-commit [hunk 2]
```diff
@@ -5,19 +5,19 @@
...
+  if [ ! -x "$(command -v nvm)" ] && [ -f ~/.nvm/nvm.sh ]; then
```
Comment (emmercm) [inline .husky/pre-commit:9]: To thread on a random diff line:

@dcramer what's your OS and your default shell? I should probably add a ShellCheck CI check to catch this kind of syntax.

Comment (dcramer) [reply .husky/pre-commit:9]: this is Ubuntu on WSL2 (though I run a current gen Macbook as well - should work there too afaik)

Comment (emmercm) [reply .husky/pre-commit:9]: Here's some proof that ShellCheck also wants me to make these changes: https://github.com/emmercm/igir/actions/runs/5909212635/job/16029534496?pr=580

File: .husky/pre-commit (hunk 1 above)
Comment (emmercm) [reply .husky/pre-commit:8]: I will have to check this out! nvm has typically been great for me, but there are definitely a lot of hacks in here for it.

**Pull request merged**